### PR TITLE
fix the download link in v5.1

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

fix the download link in v5.1

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/8955
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]
LGTM

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ summary: Use sync-diff-inspector to compare data and repair inconsistent data.
 
 This guide introduces the key features of sync-diff-inspector and describes how to configure and use this tool. To download sync-diff-inspector, use one of the following methods:
 
-+ Binary package. Click [tidb-enterprise-tools-nightly-linux-amd64](https://download.pingcap.org/tidb-enterprise-tools-nightly-linux-amd64.tar.gz) to download.
++ Binary package. Click [tidb-community-toolkit-v5.1.4-linux-amd64](https://download.pingcap.org/tidb-community-toolkit-v5.1.4-linux-amd64.tar.gz) to download.
```
Comment (shichun-0415) [inline sync-diff-inspector/sync-diff-inspector-overview.md:12]: 第 8 行 sync-diff-inspector 的仓库链接还是 master 分支的，需要改成 5.1 分支吗？

Comment (hfxsd) [reply sync-diff-inspector/sync-diff-inspector-overview.md:12]: 咱们改成分支吧？其他版本都用的分支

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**